Tests/L0/VsTest/_suite.ts

```diff
@@ -169,7 +169,7 @@ describe('VsTest Suite', function () {
     it('Vstest task with test results files filter and exclude filter', (done) => {
         setResponseFile('vstestGood.json');
         var tr = new trm.TaskRunner('VSTest');
-        tr.setInput('testAssembly', '/some/*pattern;-:/exclude/*pattern');
+        tr.setInput('testAssembly', 'some/*pattern;-:exclude/*pattern');
```

Comment:
what happens if I give filter like this `-:abc\t*dll`, `-:\abc\t*dll`, `-:abc\\t*dll` in ps version? this should be in sync with `ps` version